CHANGELOG.txt

```diff
@@ -1,3 +1,514 @@
+a85d84020 Make pytest collection warnings errors (#14832)
+303c89fea Return output of last task from task_group wrapper. (#15779)
+2b5d4e3ff Unable to trigger backfill or manual jobs with Kubernetes executor. (#14160)
+33910d6c6 Default to Celery Task model when backend model does not exist (#14612)
+2c9c55781 Stop sending Callback Requests if no callbacks are defined on DAG (#13163)
+dd78ad06e Remove redundant word (#13466)
+3909232fa Remove permissions to read Configurations for User and Viewer roles (#14067)
+41bf97469 Skip SLA check only if SLA is None (#14064)
+32f59534c Stop creating duplicate Dag File Processors (#13662)
+7f8be97da Dispose connections when running tasks with os.fork & CeleryExecutor (#13265)
+0fdc03b76 Make v1/config endpoint respect webserver expose_config setting (#14020)
+6aafee428 Print right version in airflow info command (#14560)
+e0ee91e15 Scheduler should not fail when invalid executor_config is passed (#14323)
+2366f861e Added missing return parameter in read function of FileTaskHandler (#14001)
+703452930 Stop loading Extra Operator links in Scheduler (#13932)
+a1097f6f2 Make airflow info to work with pipes (#14528)
+c71f707d2 Make airflow dags show command display TaskGroup (#14269)
+e4b8ee63b Increase the default ``min_file_process_interval`` to decrease CPU Usage (#13664)
+4b67b0b94 Remove inapplicable arg 'output' for CLI pools import/export (#13071)
+997a00971 Gracefully handle missing start_date and end_date for DagRun (#14452)
+f763b7c3a Make TaskInstance.pool_slots not nullable with a default of 1 (#14406)
+ca35bd7f7 By default PIP will install all packages in .local folder (#14125)
+4d4aa9ec2 Use template strings for string concatenation in JS code in dags.html (#13957)
+8958d125c Only compare updated time when Serialized DAG exists (#13899)
+8ecdef3e5 Audit Log records View should not contain link if dag_id is None (#13619)
+98bbe5aec Rework client-side script for connection form. (#14052)
+b4885b258 Only allow passing JSON Serializable conf to TriggerDagRunOperator (#13964)
+70ce0d814 Don't add Website.can_read access to default roles. (#13923)
+31d31adb5 Setting `max_tis_per_query` to 0 now correctly removes the limit (#13512)
+52a5ecf45 Stop Log Spamming when `[core] lazy_load_plugins` is False (#13578)
+93e4787b7 Allow PID file path to be relative when daemonize a process (scheduler, kerberos, etc) (#13232)
+18df31d5d skip entrypoint load if provider has already been loaded from local source (#13245)
+d752d92a4 Flower should be enabled for CeleryKubernetesExecutor (#13248)
+640985865 specify constraint key type & drop auto fkey referred to users tables (#13239)
+ef8f414c2 Filter DagRuns with Task Instances in removed State while Scheduling (#13165)
+
+Improvements
+""""""""""""
+
+feddc517d Add missing docstring params (#15741)
+231d104e3 Enforce READ COMMITTED isolation when using mysql (#15714)
+3614910b4 New UI: Add Timezone select (#15674)
```

Comment:
```suggestion
```

New (react) UI isn't "shipped" yet, so shouldn't include anything about it in the changelog